### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by jnsdls**
closes: DASH-545

---

## PR-Codex overview
This PR primarily updates various dependencies across multiple `package.json` files, enhancing compatibility and fixing potential issues by upgrading package versions. 

### Detailed summary
- Updated `@types/react` and `@types/react-dom` versions to `19.0.1` and `19.0.2` respectively in multiple packages.
- Upgraded `shiki` from `1.24.0` to `1.24.1` in several packages.
- Upgraded `sonner` from `^1.7.0` to `^1.7.1` in multiple packages.
- Updated `@emotion/react` and `@emotion/styled` from `11.13.5` to `11.14.0` in `packages/thirdweb`.
- Upgraded `framer-motion` from `11.13.1` to `11.13.3` in `apps/dashboard`.
- Updated `@playwright/test` from `1.49.0` to `1.49.1` in various packages.
- Updated `@dirtycajunrice/klee` from `^1.0.5` to `^1.0.6` in `apps/portal`.
- Minor updates in `pnpm-lock.yaml` for multiple dependencies to reflect new versions.

> The following files were skipped due to too many changes: `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by jnsdls** [comment]
* **#5667** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5667?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5667?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/thirdweb/package.json [hunk 1]
```diff
@@ -355,20 +355,20 @@
...
+    "msw": "2.6.8",
     "prettier": "3.3.3",
     "react": "19.0.0",
     "react-dom": "19.0.0",
-    "react-native": "0.76.4",
+    "react-native": "0.76.5",
     "react-native-aes-gcm-crypto": "0.2.2",
     "react-native-passkey": "3.0.0",
-    "react-native-quick-crypto": "0.7.8",
+    "react-native-quick-crypto": "0.7.9",
```
Comment (joaquim-verges) [inline packages/thirdweb/package.json:365]: ugh need to test this properly

**Comment by jnsdls** [comment]
### Merge activity

* **Dec 9, 7:39 PM EST**: [Graphite](https://app.graphite.dev/github/pr/thirdweb-dev/js/5667) disabled "merge when ready" on this PR due to: a merge conflict with the target branch; resolve the conflict and try again..

File: packages/thirdweb/package.json (hunk 1 above)
Comment (jnsdls) [reply packages/thirdweb/package.json]: going to not include it for now instead

**Review by joaquim-verges — APPROVED** [review]

**Pull request merged**